chapter08/e2e/tables/element.e2e-spec.ts

```diff
@@ -1,7 +1,7 @@
 import {browser, by, element, ElementFinder, ElementHelper, ProtractorBy} from 'protractor';
 import {By as webdriverBy, WebElement} from 'selenium-webdriver';
 
-describe('element methods', () => {
+fdescribe('element methods', () => {
```

Comment:
@cnishina was `fdescribe` suppose to be left in there?

Reply:
Oh that's me. Going to fix this too.